static/skin/taskbar.js

```diff
@@ -3,83 +3,86 @@ function htmlDecode(input) {
     return doc.documentElement.textContent;
 }
 
-(function ($) {
-    const root = $( `link[type='root']` ).attr("href");
-    const bookName = window.location.pathname.split(`${root}/`)[1].split('/')[0];
-
-    $( "#kiwixsearchbox" ).autocomplete({
-
-        source: `${root}/suggest?content=${bookName}`,
-        dataType: "json",
-        cache: false,
-
-        response: function( event, ui ) {
-              for(const item of ui.content) {
-                  item.label = htmlDecode(item.label);
-                  item.value = htmlDecode(item.value);
-                  if (item.path) item.path = htmlDecode(item.path);
-              }
-        },
-
-        select: function(event, ui) {
-            if (ui.item.kind === 'path') {
-                window.location.href = `${root}/${bookName}/${encodeURI(ui.item.path)}`;
-            } else {
-                $( "#kiwixsearchbox" ).val(ui.item.value);
-                $( "#kiwixsearchform" ).submit();
-            }
-        },
-    });
-
-    /* cybook hack */
-    if (navigator.userAgent.indexOf("bookeen/cybook") != -1) {
-        $("html").addClass("cybook");
-    }
-
-    if ($(window).width() < 520) {
-        var didScroll;
-        var lastScrollTop = 0;
-        var delta = 5;
-        // on scroll, let the interval function know the user has scrolled
-        $(window).scroll(function (event) {
-            didScroll = true;
+const jq = jQuery.noConflict();
```

Comment:
@mgautierfr I have already used `jQuery.noConflict` to remove conflict between 2 jquery but the problem is `jQuery.noConflict` captures the jquery that is loaded first so if the jQuery we add is loaded first it assigns that to `jq` but, if jQuery they use(zim file) is loaded first then `jq` has their version of jQuery stored in it.

I suppose using `iFrame` would be the best course of action.